docs/advice/SECURITY_CATEGORY.md

```diff
@@ -0,0 +1,48 @@
+
+# Fledger: Security Read Me
+
+**Category Advisor:** Ben Ransford (Startup Founder, currently Security engineer at Stripe, not on behalf of Stripe)
+## Why is Security important?
+
+“Security” refers to the goal of making sure systems behave as designed, even when someone tries to make them behave otherwise. Errors in security can result in serious consequences for companies, including monetary loss or reputational damage. Getting security right means being able to understand how to make systems robust.
+
+[https://www.schneier.com/blog/archives/2008/03/the_security_mi_1.html](https://www.schneier.com/blog/archives/2008/03/the_security_mi_1.html) is a short introduction to the “security mindset,” a way of thinking about systems through the eyes of an adversary that might want to make them misbehave.
+
+**Understanding security** helps engineers build systems that behave as designed.
+
+## How to use this Fledger to Practice for Security Interviews
+
+Open-source software powers the world. Security issues can happen at a package, service, or system level. Here are some key areas that you can practice with an open source project. Search for a project here [link](https://github.com/trending?since=monthly), find something you like!
+
+**Run this command** to generate the relative skill assessment to track
+```sh
+fledger build-skill-assesment SecurityCategory
+```
+
+|                        |                                                                 |                                                                                                                                                                                                                                                                                                                |
+| ---------------------- | --------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Category**           | **Description**                                                 | **Steps**                                                                                                                                                                                                                                                                                                      |
```

Comment:
I didn't go through this in super detail. It's a loose assemblage of categories along with some pretty generic suggestions! The best input will be someone in the target audience trying to work through it.